doc/architecture/decisions/0003-bulk-load-gp-data.md

```diff
@@ -0,0 +1,23 @@
+# 3. Build Image with GP Data
+
+Date: 2017-06-15
+
+## Status
+
+Accepted
+
+## Context
+
+In the early stages of ES adoption (which predates the development of ETL processes to automatically build the GP JSON data file) the GP data mas manually added to this repo. The data is in a format optimised for loading into MongoDB. Whilst Elastic Search (ES) can import documents singly in the format it takes some considerable time. By re-shaping the data to that required by the bulk import API of ES then the import can happen in a matter of seconds rather than 10's of minutes otherwise.
+
+## Decision
+
+The use of an existing data file coupled with pre-procesing (using a `jq` script) to reshape it and subsequent bulk loading it into ES was a pragmatic move
```

Comment:
Suggest rewording to `We will use the existing data file but reshape the data specifically to suit the needs of ES via jq (a lightweight JSON processing cli tool).` (or something similar).